packages/components/src/navigation/styles/navigation-styles.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { G2, UI } from '../../utils/colors-values';
+import Button from '../../button';
+import Text from '../../text';
+
+export const NavigationUI = styled.div`
+	width: 100%;
+	background-color: ${ G2.darkGray.primary };
+	color: #f0f0f0;
+	padding: 8px;
+	overflow: hidden;
+`;
```

Comment:
What do you think about adding `height: 100%` here too? It seems the likely use cases would have it occupy the entire height.

Reply:
I'm pondering a bit about how to tackle the style, especially in consideration of @ItsJonQ's [comment](https://github.com/WordPress/gutenberg/pull/25057#issuecomment-690733404).

What Q says is very valid: this component will be used in diverse ways, not just the dark-background left navigation sidebar that we are thinking of.

Maybe the 100% height (or the light/dark theme) should be the concern of the consumer?
Or maybe we could expose some preset props to make it easier? (e.g. a `theme` prop that can be light or dark, a `fullHeight` prop that makes it 100% tall)

I already have a style issue planned, and I'll add these concerns to that as well. 👍